kolibri/core/auth/management/commands/sync.py

```diff
@@ -82,6 +82,12 @@ def add_arguments(self, parser):
             action="store_true",
             help="do not create a facility and temporary superuser",
         )
+        parser.add_argument(
+            "--resync-interval",
+            type=int,
+            default=5,
```

Comment:
I think this should default to `None` so that we only do the resync in the case that we are calling it from inside a scheduled job.